app/scripts/application.js

```diff
@@ -0,0 +1,24 @@
+import AppConfig from 'scripts/config';
+import LinkOverride from 'scripts/overrides/link';
+
+let App = new Marionette.Application();
+
+App.history = function() {
```

Comment:
As I said above I don't like the idea to use fat arrow when I don't have to change the context